contracts/governance/Governor.sol

```diff
@@ -603,14 +603,15 @@ abstract contract Governor is Context, ERC165, EIP712, IGovernor, IERC721Receive
     }
 
     /**
-     * @dev See {IERC721Receiver-onERC721Received}.
+     * @dev See {IERC721Receiver-onERC721Received}(disabled if executor is a third party contract).
      */
     function onERC721Received(address, address, uint256, bytes memory) public virtual override returns (bytes4) {
+        require(_executor() == address(this), "Governor: must send to executor");
         return this.onERC721Received.selector;
     }
 
     /**
-     * @dev See {IERC1155Receiver-onERC1155Received}.
+     * @dev See {IERC1155Receiver-onERC1155Received}(disabled if executor is a third party contract).
```

Comment:
```suggestion
     * @dev See {IERC721Receiver-onERC721Received}.
     * Receiving tokens is disabled if the governance executor is other than the governor itself (eg. when using with a timelock).
```

Reply:
Hi @ernestognw, thanks for the suggestions!

Maybe is confusion or I'm understanding something wrong but, if is the `onERC1155Received` hook why 

 ``` solidity 
     * @dev See {IERC1155Receiver-onERC1155Received}.
```
should be changed by

``` solidity 
    * @dev See {IERC721Receiver-onERC721Received}.
```

Reply:
Should be 
```suggestion
     * @dev See {IERC1155Receiver-onERC1155Received}.
     * Receiving tokens is disabled if the governance executor is other than the governor itself (eg. when using with a timelock).
```

Reply:
Completely my bad. Yes, it has to be as @Amxx suggests. Same for others.